tcf_website/management/commands/fix_multiple_objects_returned.py

```diff
@@ -0,0 +1,113 @@
+from functools import reduce
+from operator import or_
+
+from django.core.management.base import BaseCommand
+from django.db.models import Case, Count, F, FloatField, Q, Sum, Value, When
+
+from tcf_website.models import CourseInstructorGrade
+
+
+class Command(BaseCommand):
+    """This fixes existing `MultipleObjectsReturned` errors"""
+
+    help = 'Fix `MultipleObjectsReturned` errors that were caused by inconsistent email data'
+
+    def handle(self, *args, **options):
+        self.verbosity = options['verbosity']
+
+        # Number of cases we want to fix
+        qs = CourseInstructorGrade.objects\
+            .exclude(course=None)\
+            .values('course', 'first_name', 'last_name')\
+            .annotate(num=Count('id'))\
+            .filter(num__gt=1)
+        if self.verbosity > 0:
+            print(qs.count())

```

Comment:
Can you add a description of what this is to the log i.e. `print(f'Number of duplicates: {qs.count()})`? Doubt it matters too much considering this is a rare one-off thing, but I think it's good practice to have complete logging.

Reply:
Done in f45bd0586dca0e7cfc008ddf272e194f660bbc75